src/Inputs/Slider.tsx

```diff
@@ -1,79 +1,456 @@
-import React from 'react';
-import styled from 'styled-components';
-import { LabelLayout, LabelLayoutProps } from '../Fragments';
+import React, {
+    useState,
+    useLayoutEffect,
+    useRef,
+    RefObject,
+    MouseEvent,
+    useMemo,
+} from 'react';
+import styled, { DefaultTheme } from 'styled-components';
+import { LabelLayout, LabelLayoutProps } from '@Layouts';
+import { Popup } from '../Containers/Popup';
+
+export interface MarkProps {
+    key: number;
+    mark: string;
+}
+
+export interface ResultValues {
```

Comment:
I think some of your variables can be more clear
at a glance I have no clue what this means or what these values are for